app/presenters/two_factor_authentication/set_up_auth_app_selection_presenter.rb

```diff
@@ -0,0 +1,19 @@
+module TwoFactorAuthentication
+  class SetUpAuthAppSelectionPresenter < SetUpSelectionPresenter
+    def method
+      :auth_app
+    end
+
+    def mfa_configuration_count
+      user.auth_app_configurations.count
+    end
+
+    def label
+      t('two_factor_authentication.two_factor_choice_options.auth_app')
+    end
+
+    def info
+      t('two_factor_authentication.two_factor_choice_options.auth_app_info')
```

Comment:
Should we start removing these strings from the other existing classes as part of the transition?